listen.go

```diff
@@ -176,6 +185,12 @@ func (app *App) Listen(addr string, config ...ListenConfig) error {
 
 		// Attach the tlsHandler to the config
 		app.SetTLSHandler(tlsHandler)
+	} else if cfg.AutoCertManager != nil {
+		tlsConfig = &tls.Config{
+			MinVersion:     tls.VersionTLS12,
```

Comment:
We should make this configurable, some people may want to do only TLS1.3

Reply:
We will address this in a separate PR/Issue

Reply:
@coderabbitai Please create an issue to make tlsconfig minversion configurable for the server listener and acme